django_redis/client/default.py

```diff
@@ -70,25 +71,32 @@ def get_next_client_index(self, write=True):
         Overwrite this function if you want a specific
         behavior.
         """
+        if tried and len(tried) < len(self._server):
+            not_tried = [i for i in range(0, len(self._server)) if i not in tried]
+            return random.choice(not_tried)
+
         if write or len(self._server) == 1:
             return 0
 
         return random.randint(1, len(self._server) - 1)
 
-    def get_client(self, write=True):
+    def get_client(self, write=True, tried=[], show_index=False):
```

Comment:
Well... it was never mutated anyway, but better safe than sorry. Changed to tuple defaults